src/vm/callingconvention.h

```diff
@@ -573,8 +573,8 @@ class ArgIteratorTemplate : public ARGITERATOR_BASE
 
         if (TransitionBlock::IsFloatArgumentRegisterOffset(argOffset))
         {
-            // Dividing by 8 as size of each register in FloatArgumentRegisters is 8 bytes.
-            pLoc->m_idxFloatReg = (argOffset - TransitionBlock::GetOffsetOfFloatArgumentRegisters()) / 8;
+            // Dividing by 8 as size of each register in FloatArgumentRegisters is 16 bytes.
```

Comment:
nit: `Dividing by 8 as size of each register in FloatArgumentRegisters is 16 bytes` should be `Dividing by 16 as size of each register in FloatArgumentRegisters is 16 bytes`

Reply:
Oops!